pkg/web_css/lib/src/_search.scss

```diff
@@ -20,7 +20,7 @@
   }
 }
 
-.search-bar > .input {
+.search-bar>.input {
```

Comment:
I think the spaces are better for readability. Although, we should rather use embedding instead...

Reply:
embedding is not the same though, right?

I'm not even sure spaces are -- probably, why am I this bad at CSS :rofl:

Reply:
Yeah, embedding is not the same, I meant:
```
.search-bar {
  > .input {
  }
}
```

Reply:
formatter removes the space though :rofl:

Reply:
but, yes I'm sold on embedding :D